new_platforms/samples/oe_policy_tool/enc/enc.c

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/bits/stdio.h>
+
+// Include the trusted oe_policy_tool header that is generated
+// during the build. This file is generated by calling the
+// sdk tool sgx_edger8r against the oe_policy_tool.edl file.
```

Comment:
shouldn't it use oe_edger8r instead of sgx_edger8r?

Reply:
Removed from here, fixed in the other repo.